api/resources/applicants.py

```diff
@@ -15,7 +15,6 @@ def _applicant_payload(applicant):
     return {
         'id': applicant.id,
         'username': applicant.username,
-        'email': applicant.email,
         'bio': applicant.bio,
         # 'skills': [{'id': skill.id, 'name': skill.name} for skill in applicant.skills],
```

Comment:
`name` here would change to `attribute` if we're fully updating the response format for the typescript needs